lumicks/pylake/kymo.py

```diff
@@ -1123,7 +1122,7 @@ def label(self):
 @dataclass(frozen=True)
 class PositionCalibration:
     unit: PositionUnit = PositionUnit.pixel
-    value: float = 1.0
+    factor: float = 1.0
```

Comment:
I have a mild preference for `scale` given that one often talks about `scale` and `offset` in the context of transforms, but I don't feel very strongly about it.